Sources/Controllers/2.0/OABaseScrollableHudViewController.mm

```diff
@@ -262,6 +261,11 @@ - (BOOL) isLandscape
     return OAUtilities.isLandscapeIpadAware;
 }
 
+- (BOOL) isLeftSidePresentation
+{
+    return NO;
```

Comment:
This will break the behavior of other custom huds in the future, please return isLandscapeIpadAware here instead